src/app/workspace/workspace-chooser/resource-chooser/resource-chooser.component.html

```diff
@@ -7,7 +7,7 @@ <h3 class="w-100 d-flex px-3" i18n="@@resource.chooser.header">
 
     <div class="px-3" *ngIf="resources.length == 0 && !canCreateResource"
          i18n="@@resource.chooser.empty">
-      There are no {resourceType, select, organization {organizations} project {projects}} and you are not allowed to
+      There are no {resourceType, select, Organization {Organizations} Project {Projects}} and you are not allowed to
```

Comment:
Isn't it going to look strange capitalized? You just need to keep the term outside the curly brackets in the original form and it does not matter what you put inside.

At the same time, you need to change the translations otherwise it will be shown as before.